tw-tasks-core/src/main/java/com/transferwise/tasks/processing/TasksProcessingService.java

```diff
@@ -212,9 +212,11 @@ protected ProcessTasksResponse processTasks(GlobalProcessingState.Bucket bucket)
         mdcService.with(() -> {
           mdcService.put(task);
           try {
-            ProcessTaskResponse processTaskResponse = grabTaskForProcessing(bucketId, task);
 
-            coreMetricsTemplate.registerTaskGrabbingResponse(bucketId, type, priority, processTaskResponse);
+            long epochMilliBeforeProcessing = System.currentTimeMillis();
```

Comment:
Would it be better to name it `epochMilliGrabForProcessing` ? describing the current state, instead of the outcome (grabbing the task Vs processing the task)

Reply:
Removed as not needed